params/version.go

```diff
@@ -58,7 +61,7 @@ func readVersionFile() (version, error) {
 	if len(vnums) != 3 {
 		return version{}, errors.New("bad version number format")
 	}
-	major, err := strconv.Atoi(string(vnums[0][:]))
+	major, err := strconv.Atoi(string(vnums[1][:])) // First byte is 'v'
```

Comment:
![image](https://user-images.githubusercontent.com/9705730/215812585-cece1c09-5c01-4ed8-bb78-46c14c3ed44e.png)
The change looks good to me. Line 55 in main 58 in hotfix looks like it should be vmeta = split[1]

Reply:
Oh, good catch!

Reply:
Fixed: 671588d5